src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -60,28 +59,53 @@ function renderEnrollmentsComponent(initialProps) {
     </Provider>,
   );
 }
+describe('Course enrollements', () => {
+  beforeEach(() => {
+    updateCourseCompleteStatusRequest.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('loads enrollments component', () => {
+    renderEnrollmentsComponent(initProps);
+    expect(screen.getByText('My courses in progress')).toBeInTheDocument();
+    expect(screen.getByText('Completed courses')).toBeInTheDocument();
+    expect(screen.getByText('Courses saved for later')).toBeInTheDocument();
+    expect(screen.getAllByText('edX Demonstration Course').length).toBeGreaterThanOrEqual(1);
+  });
 
-test('loads enrollments component', () => {
-  renderEnrollmentsComponent(initProps);
-  expect(screen.getByText('My courses in progress')).toBeInTheDocument();
-  expect(screen.getByText('Courses saved for later')).toBeInTheDocument();
-  expect(screen.getAllByText('edX Demonstration Course').length).toBeGreaterThanOrEqual(1);
-});
+  test('move to in progress action generates course status update', () => {
+    renderEnrollmentsComponent({
+      ...initProps,
+      courseRuns: {
+        ...initProps.courseRuns,
+      },
+    });
```

Comment:
unless I missed sth, could this just be 
```
renderEnrollmentsComponent(initProps)
```
?

Reply:
Yes! Historically, we were modifying the `initProps`, but now we don't need to. Good catch.

Reply:
I'm still going to use `{ ...initProps}` because I'm paranoid about not having a new object each time. ;)